cloud/azure/app-services/variables.tf

```diff
@@ -0,0 +1,513 @@
+# Global
+
+variable "environment" {
+  description = "Infrastructure environment"
+  type        = string
+}
+
+# SignalFx module specific
+
+variable "notifications" {
+  description = "Notification recipients list for every detectors"
+  type        = list
+}
+
+variable "prefixes" {
+  description = "Prefixes list to prepend between brackets on every monitors names before environment"
+  type        = list
+  default     = []
+}
+
+variable "filter_custom_includes" {
+  description = "List of tags to include when custom filtering is used"
+  type        = list
+  default     = []
+}
+
+variable "filter_custom_excludes" {
+  description = "List of tags to exclude when custom filtering is used"
+  type        = list
+  default     = []
+}
+
+variable "detectors_disabled" {
+  description = "Disable all detectors in this module"
+  type        = bool
+  default     = false
+}
+
+# Azure app services detectors specific
+
+variable "heartbeat_disabled" {
+  description = "Disable all alerting rules for heartbeat detector"
+  type        = bool
+  default     = null
+}
+
+variable "heartbeat_notifications" {
+  description = "Notification recipients list for every alerting rules of heartbeat detector"
+  type        = list
+  default     = []
+}
+
+variable "heartbeat_timeframe" {
+  description = "Timeframe for system not reporting detector (i.e. \"10m\")"
+  type        = string
+  default     = "20m"
+}
+
+# Response_time detectors
+
+variable "response_time_disabled" {
+  description = "Disable all alerting rules for response_time detector"
+  type        = bool
+  default     = null
+}
+
+variable "response_time_disabled_critical" {
+  description = "Disable critical alerting rule for response_time detector"
+  type        = bool
+  default     = null
+}
+
+variable "response_time_disabled_warning" {
+  description = "Disable warning alerting rule for response_time detector"
+  type        = bool
+  default     = null
+}
+
+variable "response_time_notifications" {
+  description = "Notification recipients list for every alerting rules of response_time detector"
+  type        = list
+  default     = []
+}
+
+variable "response_time_notifications_warning" {
+  description = "Notification recipients list for warning alerting rule of response_time detector"
+  type        = list
+  default     = []
+}
+
+variable "response_time_notifications_critical" {
+  description = "Notification recipients list for critical alerting rule of response_time detector"
+  type        = list
+  default     = []
+}
+
+variable "response_time_aggregation_function" {
+  description = "Aggregation function and group by for response_time detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".mean(by=['instance', 'azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
```

Comment:
lead to no data, correct tag is `Instance` not `instance` (seem to be the case for other detectors and other services / modules)